package org.routemaster.api.auth.domain.user.social.impl.exception;

public class SocialUserErrorDescription {

    public static final String SOCIAL_USER_ALREADY_EXIST = "User already exists.";
    public static final String SOCIAL_USER_NOT_FOUND = "User not found.";
    public static final String INVALID_PROVIDER = "Invalid Provider.";
}
